src/views/documents/utils/PrintingSummary.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div v-if="documents" class="box">
+    <h1 class="title is-1">Summary</h1>
+    <div v-for="(document, index) in documents.documents" :key="index">
+      <span @click="sendSummaryClick(index)">
```

Comment:
There is no indication (e.g. cursor) that clicking on a list item will scroll you there. Also, I suggest to us smooth scrolling to help the user undestand he is scrolling whithin the page, othewise, you might think it opened the document itself. See other review comment.
Also, maybe one should have a shortcut to go back to the top of the page?